veg2hab/main.py

```diff
@@ -79,6 +81,8 @@ def run(
 
 
 def run_1_inladen_vegkartering(params: Union[AccessDBInputs, ShapefileInputs]):
+    logging.info(f"Huidige veg2hab versie: {veg2hab.__version__}")
```

Comment:
Als je dit verplaatst naar regel 64 in `run`, hoef je het maar 1 keer aan te roepen.

Reply:
ah ja tuurlijk